uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/apis/leave-request-api_test.js

```diff
@@ -304,6 +304,26 @@ define([
       });
     });
 
+    describe('delete()', function () {
+      var idToDelete = '123';
+
+      beforeEach(function () {
+        spyOn(LeaveRequestAPI, 'sendGET');
```

Comment:
only if we want to mock the returned data. In this case I don't care about what the server returns, I only want to check if a given method has been called and with what parameter. Hence `spyOn` is enough